python/xt/test/test_common_config.py

```
@@ -0,0 +1,68 @@
+def test_version_attr_consistent_with_have():
+    cfg = Config()
+    if cfg.HAVE_K3D:
+        assert cfg.K3D_VERSION not in (None, False)
+    else:
+        assert cfg.K3D_VERSION is None
+
+
+def test_unknown_attribute_raises():
+    cfg = Config()
+    with pytest.raises(AttributeError):
+        cfg.NOT_A_VALID_ATTR
```

Ruff (0.15.17)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)